### ISSUE TITLE
Problem in predicting with cross validation papams for list of multiple models. 

### ISSUE THREAD

**Issue by sm-ak-r33** [Nixtla/mlforecast#374]
### Description

# Create a list of models
models = [
    lgb.LGBMRegressor(random_state=0, verbosity=-1),
    LinearRegression(),
    XGBRegressor(random_state=0, verbosity=0),  # XGBoost regressor
    RandomForestRegressor(random_state=0)       # Random Forest regressor
]

horizon = 292  # Size of the validation set (20% of 1460)
step_size = 234  # Step size for each fold

fcst_daily = MLForecast(
    models=models,
    freq='D',
    lags=list(range(1, 31)),
    lag_transforms={
        1: [ExpandingMean()],
        7: [RollingMean(window_size=7)]
    }
) 

# Perform cross-validation
cv_results = fcst_daily.cross_validation(
    df_daily,
    n_windows=5,
    h=horizon,
    step_size=step_size,
    fitted=True,
    refit=True,
)

 In this code, I am trying to implement the best fit from the cross validation using
 
 fcst = MLForecast.from_cv(cv)
 
 but it is not working. Is it even possible? What is the correct workflow after this?

**Comment by jmoralez** [comment]
Hey. That method is used to create an MLForecast instance from a [LightGBMCV](https://nixtlaverse.nixtla.io/mlforecast/lgb_cv.html) one. If you want to train a model with all data you can just use the `MLForecast.fit` method.

**Comment by sm-ak-r33** [comment]
Thanks. Then I assume MLForecast.fit will do the cross-validation automatically.

**Comment by jmoralez** [comment]
If you want to do hyperparameter tuning you should use the AutoMLForecast class intead ([guide](https://nixtlaverse.nixtla.io/mlforecast/docs/how-to-guides/hyperparameter_optimization.html))
